PhysicsTools/Heppy/python/analyzers/objects/JetAnalyzer.py

```diff
@@ -100,6 +101,12 @@ def process(self, event):
         if self.doJEC:
 #            print "\nCalibrating jets %s for lumi %d, event %d" % (self.cfg_ana.jetCol, event.lumi, event.eventId)
             self.jetReCalibrator.correctAll(allJets, rho, delta=self.shiftJEC, metShift=self.deltaMetFromJEC)
+
+        for delta, shift in [(1.0, "JECUp"), (0.0, ""), (-1.0, "JECDown")]:
```

Comment:
aren't we computing twice the 0.0  ?
shouldn't we make configurable the lisft of shifts and perhaps get rid of L103? we can perhaps use the first shift in the list as new default and store all the others as "corr"+shift ?
@gpetruc what do you think?